src/routes/docs/tutorials/nuxt/step-5/+page.markdoc

```diff
@@ -0,0 +1,51 @@
+---
+layout: tutorial
+title: Add navigation
+description: Add navigation to your Nuxt applicating.
+step: 5
+---
+
+In our app we want to show a navigation bar. We will add it to the `app.vue` component and use the `useUserSession` composable to show either:
+- a logout button if the user is logged in
+- or a login button if the user is not logged in
+
+Update the `app.vue` file:
```

Comment:
With what? What changed, why are we making this change? Can we maybe also add comments on the parts that changed in the code snippet?

Reply:
Comments added in code throughout the tutorial, explanations added.